### ISSUE TITLE
Stuck at Loading Screen

### ISSUE THREAD

**Issue by davidlim** [davemart-in/tablechamp-app#8]
First off, thank you for open sourcing this project! I went through your steps for setting it up, but I'm stuck at step 6 of your instructions. I am able to log in, but all I see on the browser is "Loading..."

- I opened the dev console and didn't see any javascript errors.
- I cloned the code on an internal Ubuntu server. Does the server need to be accessible from the internet?

Any ideas?  Thanks in advance.

**Comment by Menyuska** [comment]
just tried this and I am having the exact same issue. have you had any luck with fixing it?

**Comment by ChristianEmilsson** [comment]
same probelm here : /

**Comment by Menyuska** [comment]
Solved! 

I forgot to run (within the tablechamp folder):
firebase init
firebase deploy

once I did these it worked :-)

here is a link: https://firebase.google.com/docs/cli/

**Comment by davidlim** [comment]
Thanks @Menyuska for your follow up!

**Install NodeJS**
I didn't have NodeJS installed on the server. After installing NodeJS, I ran into issues with installing the firebase tools. I resolved that rebooting the server and ran the firebase-tools installation command.

**Firebase Authentication**
Next, when I ran the firebase login command, I used my Google account to authenticate and landed me on an error message because it was trying to reach localhost when I've installed TableChamp on a server. I just changed the web address from localhost to the server I installed TableChamp, and the authentication process completed.

**Firebase Initialization**
1. When I ran "firebase init", it asked me what CLI features I want to set up for this folder.
2. I chose only to set up "Realtime Database Rules" and set the default Firebase project to Table Champ.
3. I chose the database.rules.json file when asked about database setup. I chose not to overwrite the existing file.

**Firebase Deploy**
Lastly, I ran "firebase deploy" and it worked!

Thanks everyone!
